config.ru

```diff
@@ -2,5 +2,11 @@
 
 # This file is used by Rack-based servers to start the application.
 
+require 'rack-rewrite'
+
+use Rack::Rewrite do
+  r301(/.*/, 'https://www.if-me.org$&', host: 'if-me-new.org')
```

Comment:
if-me-new?

Reply:
LOL
Sorry... obviously I didn't clean up properly. That whole discussion was taking _way_ too long given I'm at work